components/DataBrowser/DataContainer.vue

```diff
@@ -58,105 +72,193 @@
       <div class="data-container">
         <FilterData />
       </div> -->
-      <br>
+      <br />
       <Dashboard />
     </span>
   </div>
 </template>
 
 <script>
-import backendQuery from '@/services/backendQuery';
+import backendQuery from "@/services/backendQuery";
+import PaginationTool from "./PaginationTool.vue";
 import SearchData from "./SearchData.vue";
 import FilterData from "./FilterData.vue";
 import DisplayData from "./DisplayData.vue";
 import Dashboard from "./Dashboard.vue";
 
 export default {
-  components: { SearchData, FilterData, DisplayData, Dashboard },
-  props: [ "category" ],
+  components: {
+    PaginationTool,
+    SearchData,
+    FilterData,
+    DisplayData,
+    Dashboard,
+  },
+  props: ["category"],
   data: () => {
     return {
-      isLoadingSearch: true,
-      totalCount: 0,
+      isLoading: true,
+      totalCount: undefined,
       currentData: [],
-      allFilterDict: {},
-      currentFilterDict: {},
-      file_type: [],
-      errorMessage: '',
-      searchContent: '',
-      relation: 'and',
-      sortBy: 'Published(asc)',
-    }
+      pageNumber: undefined,
+      limitNumber: undefined,
+      facetList: [],
+      filterDict: {},
+      searchContent: "",
+      relationType: true,
```

Comment:
Rename this variable to something else, relationType == true does not tell us that it is "and".